modules/social_features/social_group/social_group.module

```diff
@@ -1277,7 +1280,7 @@ function _social_group_get_current_group($node = NULL) {
   }
   else {
     $node = is_object($node) ? $node : \Drupal::routeMatch()->getParameter('node');
-    if (is_object($node) && !is_null($node)) {
+    if (is_object($node)) {
```

Comment:
I think it's better to add condition `if($node instanceof Nodeinterface)` instead of `is_object `